app/controller/button/DigitizeButtonController.js

```diff
@@ -305,7 +309,15 @@ Ext.define('CpsiMapview.controller.button.DigitizeButtonController', {
             },
             success: me.handleApiResponse.bind(me),
             failure: function(response) {
-                BasiGX.error(response.responseText.message);
+                me.removeLastDigitizeFeature();
+
+                var errorMessage = 'Error while requesting the API endpoint';
+
+                if (response.responseText && response.responseText.message) {
+                    errorMessage += ': ' + response.responseText.message;
+                }
+
+                BasiGX.error(errorMessage);
             }
```

Comment:
Having this also in a standalone function would be even better, but this totally non-blocking.